laser.py

```diff
@@ -1,5 +1,5 @@
 import time
-from serial import Serial
+import serial
```

Comment:
That was a mistake, sorry. You're right it makes no difference, I was troubleshooting the serial module but it turned out I was using the wrong one. I'll revert it.

In general, I prefer the "import module" style most of the time rather than "from module import Class" because it keeps the namespace cleaner and when you do create a class it 's clear where that came from without having to go back to the imports, but I use both styles and mix and match them all the time.